test/factom-event-emitter.spec.js

```diff
@@ -0,0 +1,425 @@
+const { FactomCli, FactomEventEmitter } = require('../src/factom'),
```

Comment:
They will emit once for the current block. I made the assumption that the current block is also new to the user, even though it could be some minutes old at the time that a new listener is attached.